### ISSUE TITLE
iptc.get() return a string in some cases

### ISSUE THREAD

**Issue by xDEADC0DE** [LeoHsiao1/pyexiv2#52]
Thanks for this amazing package!

I’m having a minor issue when reading the IPTC tags (or keywords) of an image, using the following code:

```
img = pyexiv2.Image(localFile)
iptc = img.read_iptc()
iptc_tags = iptc.get('Iptc.Application2.Keywords') or list()
iptc_tags.append('a new tag')
```

If the image doesn’t have tags, or if it has multiple tags then the code works fine.

However, if the image has exactly 1 tag, then `iptc.get('Iptc.Application2.Keywords')` returns a `string` instead of `list`.

Is there an option to have `iptc.get()` always return a `list`?

**Comment by LeoHsiao1** [comment]
Thank you for your feedback.
I found the reason of the problem: `Iptc.Application2.Keywords` is of type String by default. If it has multiple values, pyexiv2 converts it to List type.
In three days, I will release a new version that will proactively convert some of these fields to list types..

**Comment by xDEADC0DE** [comment]
Great! I was afraid there was something wrong with my code, or an option I hadn’t seen :)
Thanks

**Comment by LeoHsiao1** [comment]
Hi!
I just released [v2.5.0](https://github.com/LeoHsiao1/pyexiv2/releases/tag/v2.5.0).
It took me longer than expected to add other features.

**Comment by xDEADC0DE** [comment]
Great! Thanks for your help
